app/src/main/res/layout/fragment_my_books.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MyBooksFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="100sp"
+        android:layout_margin="12sp"
+        android:elevation="4dp"
+        android:background="@drawable/drop_shadow" >
+
+        <ImageView
+            android:id="@+id/book_item_image"
```

Comment:
Did you just copy the xml from the book_item? Is this just temporary until I set up the layout and listview for My Books?

Reply:
Yes! It's temporary

Reply:
Ok great, once this is merged I should be able to get some good progress. Thanks!